### DISCUSSION TITLE
Better "Add filter" bahaviour

### DISCUSSION THREAD

**Discussion by KonradHEXANT** [directus/directus#19896] [Feature Requests]
### Summary

I was trying to create a filter today. I was surprised that clicking on a label creates a new filter where it will never work because this particular filter type needs to pick some more detailed option hidden under the expand arrow. 

### Basic Example

![image](https://github.com/directus/directus/assets/17060750/8a5c0194-9e49-4e62-847b-19f1cc71c565)

### Motivation

This is very bad UX if you can do something wrong and there is no response to your action at all. It simply doesn't work.

### Detailed Design

If there is some more detailed selection to be made, a label should EXPAND that filter type rather than create an unusable entry without any warning. 

### Should Have:
- Clicking on labels expands options if available. Only if no options are there, then create a filter accordingly.

### Could Have:
- If the above is not possible there should be some feedback the chosen filter won't work.

### Drawbacks

---

### Alternatives

---

### Adoption Strategy

---

**Comment by br41nslug** [comment]
> I was surprised that clicking on a label creates a new filter where it will never work

The filter does work but it expects the UUID primary key of the folder you're linking to.
Having said that it probably should show an error or warning, perhaps closing this in favor of your issue about not saving or no warning would be the way to go.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
